CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ per cluster.
 - Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
 - Add prompt for availability zone in pcluster configure automated subnets creation.
 - Remove support for CentOS8.
+- Add configuration HeadNode.Imds.Secured to enable/disable restricted access to IMDS.
```

Comment:
Is it part of your task to add the default values in the output of the pcluster configure command?

Reply:
no. that will be handled via a separate task because it's still pending a decision.